### DISCUSSION TITLE
Welcome to discussions about "The Science of Functional Programming"!

### DISCUSSION THREAD

**Discussion by winitzki** [winitzki/sofp#27] [General]
## 👋 Welcome!
 Discussions is a place to connect with other members of our community:
  * Ask questions you’re wondering about.
  * Share ideas about the project.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

# About SOFP

This is a book about the practically useful parts of the theory of functional programming. Please feel free to initiate discussions about specific issues with the book's text, questions about things that are not clear, or suggestions for improving the book.

**Comment by allenjhan** [comment]
Hi there! I just want to say I really appreciate the work you've done to present a solid technical foundation for people who want to learn functional programming in a rigorous fashion. I have just had some time open up in my schedule and can devote a few months to read this book from cover to cover. I probably will not have time to do all the exercises, but do have time to read every line of the text, and offer any suggestions if I have any. Really looking forward to the completion of chapter 13!

**Comment by winitzki** [comment]
Hi! It is great if you can read the book and send me any comments and suggestions.

**Comment by jmgimeno** [comment]
I've found this discussion page and I wonder which workflow do you prefer for erratas:

- create an issue
- comment on the discussion channel

BTW, I reported an issue suggesting a bigger font size, maybe in this case, it's better placed in the discussion.

So, which workflow do you prefer?

**Comment by winitzki** [comment]
Errata and concrete suggestions ("equations in the book must be less green") should be handled via issues or PRs.

Discussions are when there is something more philosopical to discuss ("functional programming is not what you say it is").

**Comment by ThomasSteindl** [comment]
What is the current status of this project? I'd be interested in the final version also having a look on some sample pages.

**Reply by winitzki** [reply]
The current status is that I am writing Chapter 13. This is the last chapter left to be written, all other chapters are "ready".

You can download the current stable PDF version from github's release pages. Click on the icon "Build PDF passing" and you will see links to the latest build. You will need to be logged in to github.

Let me know if you have any comments!

**Reply by yisraelU** [reply]
I think the link for the downloads expired

**Reply by winitzki** [reply]
Thank you, I will renew the link.
